docs/benchmark/ftlib_benchmark.md

```diff
@@ -0,0 +1,171 @@
+## Benchmark of FTlib AllReduce
+
+We perform experiments to test the performance of FTlib AllReduce in
```

Comment:
We perform experiments to test the performance of FTlib AllReduce in ElasticDL
=>
We benchmarked FTlib AllReduce with ElasticDL